_posts/2022-08-04-decade-innovation.md

```diff
@@ -0,0 +1,182 @@
+Storage, was supplemented by support for Amazon Glue as metastore. We also
+constantly added support for different file formats in these systems, and
+improved performance for ORC, Parquet, Avro, and others.
+
+The initial idea to support other data sources, led to connectors for over a
+dozen other databases including relational systems such
+[PostgreSQL](https://www.postgresql.org/),
+[Oracle](https://www.oracle.com/database/), [SQL
+Server](https://www.microsoft.com/en-us/sql-server), and many others. We also
+gained support for [Elasticsearch](https://www.elastic.co/elasticsearch/) and
+[OpenSearch](https://www.opensearch.org/), [MongoDB](https://www.mongodb.com/),
+[Apache Kafka](https://kafka.apache.org/), and other systems that traditionally
+are not available to query with SQL. Trino unlocks completely new use cases for
+these systems.
+
+The wide range of supported systems includes traditional data lakes and data
+warehouses. With the emerging new table formats and the related Trino
+connectors, our project is a powerful tool to run your lakehouse system. [Delta
+Lake](https://delta.io/) and [Apache Iceberg](https://iceberg.apache.org/)
+connectors are already capable of full read and write operations and include
+numerous other features. A [Apache Hudi](https://hudi.apache.org/) connector is
+in the works and coming soon.
+
+We also have robust and widely used connectors for real-time analytics systems
+like [Apache Pinot](https://pinot.apache.org/), [Apache
+Druid](https://druid.apache.org/) and [Clickhouse](https://clickhouse.com/),
+that are constantly improved by the community.
+
+## Query processing and performance
+
+Last but not least, these queries also need to be processed. From the start high
+efficiency and low latency were a core design goal, and with features like
+native compilation the delivered performance superseded other systems. Over the
+years our query analyzer and planner was supplemented by more and more
+sophisticated algorithms and features. Connectors learned to retrieve and manage
+table statistics, the optimizer was created and morphed into a [cost-based
+optimizer]({% post_url 2019-07-04-cbo-introduction %}), and we added further
+improvements that benefit query processing performance. We added dynamic
+filtering, [dynamic partition pruning]({% post_url
+2020-06-14-dynamic-partition-pruning %}), predicate pushdown, join pushdown,
+aggregate function pushdown and numerous others. Each of these improvements was
+also finely tuned, and runs in production with huge workloads providing us more
+data on how to improve next.
+
+One large pivot we recently added was the addition of [fault-tolerant query
+execution mode]({% post_url 2022-05-05-tardigrade-launch %}). Queries no longer
+die from a minor failure with this feature enabled. Parts of the execution can
+be retried and query processing can proceed. Trino is moving on from the best
+analytics engine to be the best query engine overall!
```

Comment:
> the best query engine overall!

This is claiming too much -- Trino is not trying to become, say, an OLTP query engine